build/ci/jenkins_pipeline_build

```diff
@@ -1,4 +1,11 @@
 pipeline {
+    parameters {
+        string(name: 'IMAGE_VERSION', defaultValue: "1.7.0")
```

Comment:
@zingero could you make sure the CSI-3091 **successor** ticket description will be updated accordingly?

Reply:
Done

Reply:
thanks